.gitlab-ci.yml

```
@@ -41,16 +42,17 @@ test:
 
 test:modules-artifact-gen:
   stage: test
-  image: python:3.10
+  image: ubuntu:22.04
   before_script:
+    - apt update && apt install -yy $(cat support/modules-artifact-gen/tests/deb-requirements.txt)
     # mender-artifact
-    - curl https://downloads.mender.io/mender-artifact/master/linux/mender-artifact
-      -o /usr/local/bin/mender-artifact
-    - chmod +x /usr/local/bin/mender-artifact
+    - curl -fsSL https://downloads.mender.io/repos/debian/gpg | tee /etc/apt/trusted.gpg.d/mender.asc
+    - echo "deb [arch=$(dpkg --print-architecture)] https://downloads.mender.io/repos/debian ubuntu/jammy/stable main" | tee /etc/apt/sources.list.d/mender.list
```

Sorry I was too quick with approving! This should be `experimental` to install the master package (as we used to do)
```suggestion
    - echo "deb [arch=$(dpkg --print-architecture)] https://downloads.mender.io/repos/debian ubuntu/jammy/experimental main" | tee /etc/apt/sources.list.d/mender.list
```